apps/api/src/server/applications/sector/__tests__/get-sectors.test.js

```diff
@@ -89,3 +91,87 @@ describe('Get sectors', () => {
 		});
 	});
 });
+
+test('test if sector cache is working', async () => {
+	nodeCache.flushAll();
+
+	const cacheSector = {
+		id: 3,
+		name: 'cache test name',
+		abbreviation: 'CC',
+		displayNameEn: 'cache test name en',
+		displayNameCy: 'cache test name cy'
+	};
+
+	setCache('', [cacheSector]);
+
+	const response = await request.get('/applications/sector');
+
+	expect(response.status).toEqual(200);
+	expect(response.body).toEqual([
+		{
+			name: cacheSector.name,
+			abbreviation: cacheSector.abbreviation,
+			displayNameEn: cacheSector.displayNameEn,
+			displayNameCy: cacheSector.displayNameCy
+		}
+	]);
+});
+
+const sectorName = 'Test1';
+
+test('test if cache is working with specific string', async () => {
+	nodeCache.flushAll();
```

Comment:
If we have repeating things like `nodeCache.flushAll()` when preparing tests, I would recommend using a `beforeEach` to avoid repeating code